### ISSUE TITLE
camsoda error line 80

### ISSUE THREAD

**Issue by alucard13s** [beaston02/CamsodaRecorder#5]
hi,
Now i'm not anymore banned but the script don't want to launch.
When i launch python script i have an error about line 80 
![camsoda new error](https://user-images.githubusercontent.com/30068831/34909023-11122bce-f892-11e7-806d-c2569a8c2137.PNG)
And i cant intall pip 3 request, i have these messages
![capture request 1](https://user-images.githubusercontent.com/30068831/34909032-3329b01a-f892-11e7-841c-d002ad0eb747.PNG)
![capture request 2](https://user-images.githubusercontent.com/30068831/34909033-355dab20-f892-11e7-9340-a2b3e93ecb7e.PNG)
I don't know what i have to do to making it work can you help me !
Thanks in advance.

**Comment by ls117** [comment]
im pretty sure that because of site changes this recorder has been obsolete since at least december. i had to resort to rtmpdumphelper/k-meleon, which sucks because k-meleon likes to crash.

**Comment by ghost** [comment]
Is that on windows 
@ls117

**Comment by ls117** [comment]
yeah but its not as capable as the other recorders. before the october bullshit camsoda pulled i mean.

**Comment by beaston02** [comment]
Sorry for the slow updates. Ive been out of town a lot, and havent had much time to work on this while in town. 

I just created a new branch called "test" and uploaded a script which should be working. 

There are some changes to it other than just the fix, so make sure to set the options in the config file. 

@ls117 This version does not record private/ticket streams, which I know you have been looking for a solution for. Feel free to PM me on reddit if you have found any updates yourself. 

Im still very busy, so I cant promise a quick turn around on any issues, but I wanted to get this updated and see if/how it works. I have only done minimal testing myself.

**Comment by alucard13s** [comment]
thanks the test version work but why my record don't have any sound ? I tested with FFMPEg or livestrimer sound don't work (i don't know why but i can' record with streamlink). Is It from my FFMPEG version or it's from the script ? In the normal version when it worked, i never had sound issue. Thanks in avance for your help.
